apps/storybook/stories/components/radio/Radio.stories.tsx

```diff
@@ -0,0 +1,221 @@
+    table: {
+      type: { summary: "string" },
+    },
+    control: { type: "text" },
+  },
+  isFocusable: {
+    description: "If true, and isDisabled is true, the radio will remain focusable but not interactive",
+    table: {
+      type: { summary: "boolean" },
+    },
+    defaultValue: { summary: "false" },
+    control: { type: "boolean" },
+  },
+  isChecked: {
+    description:
+      "If true, the radio will be set as checked. You need to pass onChange to update its value (since it is controlled)",
+    table: {
+      type: { summary: "boolean" },
+    },
+    defaultValue: { summary: "false" },
+    control: { type: "boolean" },
+  },
+  isInvalid: {
+    description: "If true, the radio will be invalid. Sets 'aria-invalid' to true",
+    table: {
+      type: { summary: "boolean" },
+    },
+    defaultValue: { summary: "false" },
+    control: { type: "boolean" },
+  },
+  isReadOnly: {
+    description: "If true, the radio will be read-only",
+    table: {
+      type: { summary: "boolean" },
+    },
+    defaultValue: { summary: "false" },
+    control: { type: "boolean" },
+  },
+  isRequired: {
+    description: "If true, the radio button will be required. Sets 'aria-required' to true",
+    table: {
+      type: { summary: "boolean" },
+    },
+    defaultValue: { summary: "false" },
+    control: { type: "boolean" },
+  },
+  name: {
+    description: "The name of the input field in a radio",
+    table: {
+      type: { summary: "string" },
+    },
+    control: { type: "text" },
+  },
+  onChange: {
+    description: "The function that is ran when the state of the radio changes",
+    table: {
+      type: { summary: "callback()" },
+    },
+    defaultValue: { summary: "none" },
+  },
+  colorScheme: {
+    description: "Color of the radio",
+    table: {
+      type: {
+        summary:
+          "whiteAlpha | blackAlpha | gray | red | orange | yellow | green | teal | blue | cyan | purple | pink | linkedin | facebook | messenger | whatsapp | twitter | telegram",
+      },
+      defaultValue: { summary: "blue" },
+    },
+    options: [
+      "whiteAlpha",
+      "blackAlpha",
+      "gray",
+      "red",
+      "orange",
+      "yellow",
+      "green",
+      "teal",
+      "blue",
+      "cyan",
+      "purple",
+      "pink",
+      "linkedin",
+      "facebook",
+      "messenger",
+      "whatsapp",
+      "twitter",
+      "telegram",
+    ],
+    control: { type: "select" },
+  },
+};
+
+// const radioGroupArgTypes = {};
+
+export default meta;
+type Story = StoryObj<typeof KvibRadio>;
+
+export const Radio: Story = {
+  argTypes: radioArgTypes,
+  args: { onChange: undefined },
+  render: (args) => <KvibRadio {...args}>Alternativ</KvibRadio>,
+};
+
+// export const RadioGroup: Story = {
+//   argTypes: radioGroupArgTypes,
+//   args: {},
+//   render: (args) =>
+//     <KvibRadioGroup {...args}>
+//       <KvibRadio>Radioknapp</KvibRadio>
+//     </KvibRadioGroup>,
+// };
+
+export const Color: Story = {
```

Comment:
For UU-testene sin del hadde det vært fint å legge til alle farger her, slik at vi automatisk får kontrastsjekket alle farger. Da bør du også skifte retning på visningen (fjerne direction={column} på KvibStack) for å spare plass.